typescript/cli/src/utils/fork.ts

```diff
@@ -0,0 +1,101 @@
+import { providers } from 'ethers';
+
+import { ChainName, MultiProvider } from '@hyperlane-xyz/sdk';
+
+import { logGray, logGreen, logYellow } from '../../logger.js';
+
+import { ENV } from './env.js';
+
+const ENDPOINT_PREFIX = 'http://';
+const DEFAULT_ANVIL_ENDPOINT = 'http://127.0.0.1:8545';
+
+export enum ETH_RPC_METHODS {
+  BLOCK_NUMBER = 'eth_blockNumber',
+}
```

Comment:
is this unused? I assume we are just using the ethers RPC client everywhere

Reply:
Yep, this was within the scope of my cleanup post-wip. Now removed